packages/ffe-form/less/radio-button.less

```diff
@@ -139,12 +139,11 @@
         box-shadow: none;
     }
 
-    [aria-invalid='true'] &:checked + .ffe-radio-button::after {
-        border-color: var(--ffe-g-error-color);
-    }
-
-    @media (hover: hover) and (pointer: fine) {
```

Comment:
Fjernade denne hover da jag ikka så noen effect. Antar den var der for spesifitet en gang i tiden eller noe?

Reply:
Ser sånn ut. Jeg klarte heller ikke å gjenskape denne.

Reply:
Skrev om lite. Fjerner så mye white og black som mulig slik att man kan bruke disse på andra bakgrunder også  med css custom properties. Gjorde det efter att Helene opdaget en darkmode mangel.